utkarsh component/alert/alert.css

```diff
@@ -0,0 +1,367 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Thai+Looped:wght@300&display=swap');
+
+@import url('https://cache-ui.netlify.app/css/components/alert.css');
+
+
+
+body{
+    font-family: 'IBM Plex Sans Thai Looped', sans-serif;
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+background-color:  #31353a;
+height: 100%;
+
+}
+
+* {
+   box-sizing: border-box;
+--primary-color: #ecfeff ;
+--secondary-color: #4908fa;
+}
+nav{
+  background: #0082e6;
+  height: 80px;
+  width: 100%; 
+}
+label.logo{
+  color: #fff;
+  font-size: 35px;
+  line-height: 80px;
+  padding: 0 100px;
+  font-weight: bold;
+}
+nav .main-list{
+  float: right;
+  margin-right: 20px;
+}
+nav .main-list li{
+  display: inline-block;
+  line-height: 30px;
+  /* margin: 50px 0; */
+  top: 2px;
+}
+nav .main-list li a{
+  color: #fff;
+  font-size: 17px;
+  text-transform: uppercase;
+}
+a.active,a:hover{
+  background: #1b9bff;
+  transition: .5s;
+}
+.checkbtn{
+  font-size: 30px;
+  color: #fff;
+  float: right;
+  line-height: 80px;
+  margin-right: 40px;
+  cursor: pointer;
+  display: none;
+}
+#check{
+  display: none;
+}
+@media (max-width: 480px){
+  label.logo{
+    font-size: 30px;
+    padding-left: 50px;
+  }
+  nav  li a{
+    font-size: 16px;
+  }
+}
+@media (max-width: 858px){
+  .checkbtn{
+    display: block;
+  }
+  .main-list{
+    position: fixed;
+    width: 100%;
+    height: 100vh;
+    background: #2c3e50;
+    top: 80px;
+    left: -100%;
+    text-align: center;
+    transition: all .5s;
+  }
+  nav .main-list li{
+    display: block;
+  }
+  nav .main-list a{
+    font-size: 20px;
+  }
+  a:hover,a.active{
+    background: none;
+    color: #0082e6;
+  }
+  #check:checked ~ .main-list{
+    left: 0;
+  }
+}
+
+.header{
+    background: #ee5253;
+    width: 100%;
+    height: 120px;
+    position: absolute;
+    z-index: 10;
+    top: 0;
+}
+p{
+    color: #fff;
+}
+.main{
+    height: 100%;
+    margin-top: 60px;
+    padding: 10px 50px;
+}
+#sidebarMenu{
+    height: 200%;
+    position: fixed;
+    left: 0;

```

Comment:
Please avoid using an absolute unit like px..use rem instead for better responsiveness